ft6336/ft6336.go

```diff
@@ -5,28 +5,29 @@
 package ft6336
 
 import (
-	"machine"
-
 	"tinygo.org/x/drivers"
 	"tinygo.org/x/drivers/internal/legacy"
+	"tinygo.org/x/drivers/internal/pin"
 	"tinygo.org/x/drivers/touch"
 )
 
 // Device wraps FT6336 I2C Self-Capacitive touch
 type Device struct {
-	bus     drivers.I2C
-	buf     []byte
-	Address uint8
-	intPin  machine.Pin
+	bus           drivers.I2C
+	buf           []byte
+	Address       uint8
+	configurePins func()
 }
 
 // New returns FT6336 device for the provided I2C bus using default address.
-func New(i2c drivers.I2C, intPin machine.Pin) *Device {
+func New(i2c drivers.I2C, intPin pin.Input) *Device {
```

Comment:
Interrupt pin is not used in the driver? I believe we can just drop it altogether.

Reply:
This can be a follow up